odoo/api.py

```diff
@@ -797,6 +797,7 @@ def remove_to_compute(self, field, records):
     @contextmanager
     def norecompute(self):
         """ Delay recomputations (deprecated: this is not the default behavior). """
```

Comment:
Should probably fix the docstring while at it? I expect the "not" should be "now" (it makes no sense as-is)